test/chplenv/autocomplete/autocomplete.chpl

```diff
@@ -0,0 +1,27 @@
+use Spawn, Sys;
+
+/* This test runs a script to generate a bash autocomplete script from
+   the output of `chpl --help --devel`.  It diffs the generated script
+   against the version in $CHPL_HOME/util/chpl-completion.bash.  If this
+   test produces any output it probably means that an option has been
+   added or removed from `chpl --help --devel`, so the autocomplete
+   script should be regenerated.
+ */
+
+var homeC: c_string;
+sys_getenv("CHPL_HOME", homeC);
```

Comment:
Any reason to not just use the builtin `CHPL_HOME`? (https://chapel-lang.org/docs/latest/builtins/internal/ChapelEnv.html#ChapelEnv.CHPL_HOME)

Reply:
No. Good call - I'll update that.